package.json

```diff
@@ -91,7 +91,7 @@
     "react-bootstrap": "~0.33.0",
     "react-codemirror2": "^8.0.0",
     "react-dom": "~16.13.1",
-    "react-markdown": "6.0.0",
+    "react-markdown": "6.0.3",
```

Comment:
Curious we don't have `~` in the first place - probably a better change to open this up.